.github/workflows/build-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python: ["3.8", "3.9", "3.10"]
+        python: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
why are we not removing `3.8` then?

Reply:
We talked about this in the call yesterday (sorry you couldn't make it). My motivation at least was to get pandas to v2.1. However, we do need the otherml tests (they provide the 'real' tests of `adversarial`) and those, somewhere, have a v2.0 pin.

Since Python 3.8 does still have another year of support, I'm not in a rush to drop it from Fairlearn.

Reply:
By the time we do the next release, we'll have python 3.12 there. Do we want to support 5 different python versions?